PVList.md

```diff
@@ -0,0 +1,344 @@
+    ProdSN-Cte <br>
+    ProdPN-Cte <br>
+    PowerCtl-Sel <br>
+<br>
+</details>
+
+### `$(AREA):$(DIS)-MCHClkSw:`
+<details>
+<summary>sensors</summary>
+    TempDCDC{-Mon|Prs-Cte} <br>
+    TempDCDCMax{-Mon|Prs-Cte} <br>
+    TempMux{-Mon|Prs-Cte} <br>
+    TempMuxMax{-Mon|Prs-Cte} <br>
+    Volt3V3{-Mon|Prs-Cte} <br>
+    Volt12V{-Mon|Prs-Cte} <br>
+<br>
+</details>
+<details>
+<summary>FRU</summary>
+    Prs-Cte <br>
+    FruId-Cte <br>
+    BoardManuf-Cte <br>
+    BoardName-Cte <br>
+    BoardSN-Cte <br>
+    BoardPN-Cte <br>
+    ProdManuf-Cte <br>
+    ProdName-Cte <br>
+    ProdSN-Cte <br>
+    ProdPN-Cte <br>
+    PowerCtl-Sel <br>
+<br>
+</details>
+
+### `$(AREA):$(DIS)-PciHub:`
+<details>
+<summary>sensors</summary>
+    TempDCDC{-Mon|Prs-Cte} <br>
+    TempDCDCMax{-Mon|Prs-Cte} <br>
+    TempSwitch{-Mon|Prs-Cte} <br>
+    TempSwitchMax{-Mon|Prs-Cte} <br>
+    TempPCB{-Mon|Prs-Cte} <br>
+    TempPCBMax{-Mon|Prs-Cte} <br>
+    Volt0V9{-Mon|Prs-Cte} <br>
+    Volt0V9Analog{-Mon|Prs-Cte} <br>
+    Volt1V8{-Mon|Prs-Cte} <br>
+    Volt3V3{-Mon|Prs-Cte} <br>
+    Volt12V{-Mon|Prs-Cte} <br>
+<br>
+</details>
+<details>
+<summary>FRU</summary>
+    Prs-Cte <br>
+    FruId-Cte <br>
+    BoardManuf-Cte <br>
+    BoardName-Cte <br>
+    BoardSN-Cte <br>
+    BoardPN-Cte <br>
+    ProdManuf-Cte <br>
+    ProdName-Cte <br>
+    ProdSN-Cte <br>
+    ProdPN-Cte <br>
+    PowerCtl-Sel <br>
+<br>
+</details>
+
+### `$(AREA):$(DIS)-RTM_CPU:`
+<details>
+<summary>sensors</summary>
+    TempPCB{-Mon|Prs-Cte} <br>
+    TempPCBMax{-Mon|Prs-Cte} <br>
+    TempDCDC{-Mon|Prs-Cte} <br>
+    TempDCDCMax{-Mon|Prs-Cte} <br>
+    TempComExCPU{-Mon|Prs-Cte} <br>
+    TempComExPCH{-Mon|Prs-Cte} <br>
+    TempComExDDR{-Mon|Prs-Cte} <br>
+    Volt1V0{-Mon|Prs-Cte} <br>
+    Volt1V8{-Mon|Prs-Cte} <br>
+    Volt3V3{-Mon|Prs-Cte} <br>
+    Volt5V{-Mon|Prs-Cte} <br>
+    VoltVBAT{-Mon|Prs-Cte} <br>
+    Volt12VInt{-Mon|Prs-Cte} <br>
+<br>
+</details>
+<details>
+<summary>FRU</summary>
+    Prs-Cte <br>
+    FruId-Cte <br>
+    BoardManuf-Cte <br>
+    BoardName-Cte <br>
+    BoardSN-Cte <br>
+    BoardPN-Cte <br>
+    ProdManuf-Cte <br>
+    ProdName-Cte <br>
+    ProdSN-Cte <br>
+    ProdPN-Cte <br>
+    PowerCtl-Sel <br>
+<br>
+</details>
+
+## Cooling Unit
+
+### `$(AREA):$(DIS)-CU{Top|Bot}:`
+<details>
+<summary>sensors</summary>
+   TempLM75{-Mon|Prs-Cte} <br>
+   TempLM75_2{-Mon|Prs-Cte} <br>
+   Volt3V3{-Mon|Prs-Cte} <br>
+   Volt12V{-Mon|Prs-Cte} <br>
+   Volt12V_1{-Mon|Prs-Cte} <br>
+   SFan1{-Mon|Prs-Cte} <br>
+   SFan2{-Mon|Prs-Cte} <br>
+   SFan3{-Mon|Prs-Cte} <br>
+   SFan4{-Mon|Prs-Cte} <br>
+   SFan5{-Mon|Prs-Cte} <br>
+   SFan6{-Mon|Prs-Cte} <br>
+<br>
+</details>
+<details>
+<summary>FRU</summary>
+    Prs-Cte <br>
+    FruId-Cte <br>
+    BoardManuf-Cte <br>
+    BoardName-Cte <br>
+    BoardSN-Cte <br>
+    BoardPN-Cte <br>
+    ProdManuf-Cte <br>
+    ProdName-Cte <br>
+    ProdSN-Cte <br>
+    ProdPN-Cte <br>
+    PowerCtl-Sel <br>
+    Pwr <br>
```

Comment:
Isn't this missing a standard suffix?

Reply:
Yes, thanks! It's defined as an `-Mon`.

https://github.com/lnls-dig/ipmi-mgr-epics-ioc/blob/ef8403fd679da8e7df936fabf8773f054d031fd4/ipmiMgrApp/Db/power_mngr.template#L1